src/Discord.Net.Rest/Entities/Guilds/RestGuild.cs

```diff
@@ -726,11 +784,24 @@ public Task<RestCategoryChannel> CreateCategoryChannelAsync(string name, Action<
         /// <param name="options">The options to be used when sending the request.</param>
         /// <exception cref="ArgumentNullException"><paramref name="name" /> is <see langword="null"/>.</exception>
         /// <returns>
-        ///     The created category channel.
+        ///     The created forum channel.
```

Comment:
Rename `category` on line 780 to `forum`, like here

Reply:
ah yup, nice catch